app/authenticators/mu-semtech.js

```diff
@@ -1 +1,4 @@
+// For some reason this import fails from inside the `addon` code, unless it is included somewhere
+// in the `app` directory, so why not here?
+import 'ember-simple-auth/authenticators/base';
```

Comment:
Not sure what this exactly means @piemonkey? Something we should investigate further before merging?

Reply:
What I mean is, without this `import` line, using the add-on in a project fails. The error that it gives is that addon/authenticators/mu-semtech.js cannot import something that does not exist (or that the object it's importing does not exist when it is supposed to be used, I forget).

I know that for our addon to be usable in the transpiled and combined code, we need to export it from the `app` dir, but I don't know enough about what optimisations Ember is attempting to perform, which is presumably leading to imports or dependencies being stripped.

So, with this line here, it works, but I don't know why it doesn't work without the line... To figure out where it's going wrong seems like it would take a lot of digging into the processing that Ember is doing.

Reply:
Moving ember-simple-auth from `devDependencies` to `dependencies` in `package.json` resolved the issue. It's not needed to import in `app/authenticators/mu-semtech.js` anymore.